### PR TITLE
Issue #4845: Removed checkstyle dependencies from emptyLineSeparator …

### PR DESCRIPTION
**Pull request by TanayMorakhia**
part of #4845

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java [hunk 1]
```diff
@@ -20,9 +20,6 @@
 import java.util.Collections;
 /* no violation: block comment after token*/
 
-import com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator.InputEmptyLineSeparator;
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java:23]: We need to replace this with another import and trailing single line comment

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java [hunk 2]
```diff
@@ -20,9 +20,6 @@
 import java.util.Collections;
 /* no violation: block comment after token*/
 
-import com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator.InputEmptyLineSeparator;
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java:23]: Same as above

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Items:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java (hunk 1 above)
Comment (TanayMorakhia) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java:23]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java (hunk 2 above)
Comment (TanayMorakhia) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java:23]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java [hunk 3]
```diff
@@ -14,13 +14,12 @@
 import java.io.Serializable; // violation ''import' should be separated from previous line.'
 import java.util.ArrayList; /*no violation: trailing comment*/
 import java.util.HashMap;
-import java.util.List;
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java:17]: Please restore this.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java [hunk 4]
```diff
@@ -14,13 +14,12 @@
...
 import java.util.Map;
 import java.util.concurrent.Callable;
 import java.util.Collections;
 /* no violation: block comment after token*/
 
-import com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator.InputEmptyLineSeparator;
+import java.util.List;
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java:22]:
```suggestion
import java.io.PrintWriter;
```

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java [hunk 5]
```diff
@@ -14,13 +14,12 @@
 import java.io.Serializable; // violation ''import' should be separated from previous line.'
 import java.util.ArrayList; /*no violation: trailing comment*/
 import java.util.HashMap;
-import java.util.List;
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java:17]: Please restore this.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java [hunk 6]
```diff
@@ -14,13 +14,12 @@
...
 import java.util.Map;
 import java.util.concurrent.Callable;
 import java.util.Collections;
 /* no violation: block comment after token*/
 
-import com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator.InputEmptyLineSeparator;
+import java.util.List;
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java:22]:
```suggestion
import java.io.PrintWriter;
```

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Items:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java (hunk 5 above)
Comment (TanayMorakhia) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator2.java]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java (hunk 4 above)
Comment (TanayMorakhia) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java (hunk 3 above)
Comment (TanayMorakhia) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java]: done

**Pull request merged**